src/Components/NavBar/NavItem.tsx

```diff
@@ -75,7 +81,6 @@ export const NavItem: React.FC<NavItemProps> = ({
 
   return (
     <Box
-      position="relative"
       onMouseEnter={() => toggleHover(true)}
       onMouseLeave={() => toggleHover(false)}
```

Comment:
We should be careful here in terms of fast UI responsiveness since most nav items are smaller and vertically oriented. That said, this addition might improve _all_ of the nav items, but we also might want to make this timeout configurable from the parent and only apply it to the new `Artworks` mega nav. I've noticed in the past having trouble "catching" even the small navs, so a timeout might be just the solution.